stats/src/styles/globals.css

```
@@ -94,3 +94,17 @@ body.overlay-mode #root {
 .animate-fade-in {
   animation: fadeSlideIn 0.25s ease-out;
 }
+
+/* Indeterminate progress sweep for the global delete indicator */
+@keyframes indeterminateSweep {
```

Stylelint (17.14.0)

[error] Expected keyframe name "indeterminateSweep" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)